src/Indigo/Contracts/Holdings/Types.hs

```diff
@@ -92,6 +92,27 @@ dummyMeta = TokenMeta [mt|KekToken|] [mt|Kek|] [mt|Ququareq|]
 
 type Storage = ML.StorageSkeleton StorageFields
 
+instance TypeHasDoc StorageFields where
+  typeDocName _ = "Holdings storage fields"
+  typeDocMdDescription =
+    "Additional contract fields that defines current contract state. \
+    \It stores meta-information abou token. See `TokenMeta` type. \
+    \Information about the `owner` and the current `admin` of the contract. \
+    \Additionally it stores optional address for currently used Safelist \
+    \contract, `paused`, `transferable` flags and `totalSupply` amount."
+
+instance TypeHasDoc (ML.StorageSkeleton StorageFields) where
+  typeDocName _ = "Holdings storage"
+  typeDocMdDescription =
+    "Storage used in ManagedLedger-like contracts. \
+    \The main field of this skeletion is `ledger`. It is a `big_map` that \
+    \maps each registered address to its balance and approvals amounts. \
+    \Apart from `ledger` is stores `fields` that can vary for different contracts. \
+    \See `StorageFields` types."
```

Comment:
That's true, the problem is that autodoc doesn't produce nice documentation for polymorphic types. To be specific, it should show some specific "example" (e.g. `List` or `Maybe`). I thought that it wouldn't be nice if the storage description will have an `example` instead of specific `StorageSkeleton` implementation.

Reply:
Apparently, I should remove '`fields` that can vary for different contracts.' in this case

Reply:
Well, or you can additionally say "and fixed for this specific contract". And then `s/types/type`.